data/kubemacpool/003-secret.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: Secret
```

Comment:
I believe Secrets on OpenShift require privileged pods, hence we would need to create an SCC. Did you test this on OpenShift?

Reply:
don't need any special scc.

oc get po
```
kubemacpool-system                  kubemacpool-mac-controller-manager-0               1/1       Running   1          1m        10.128.0.47      node01    <none>
```

oc logs
```
{"level":"info","ts":1554124103.0487776,"logger":"Main","msg":"Setting up client for manager"}
{"level":"info","ts":1554124103.1602395,"logger":"Main","msg":"kubevirt was not found in the cluster start a watching process"}
{"level":"info","ts":1554124103.160369,"logger":"Main","msg":"Setting up manager"}
{"level":"info","ts":1554124103.3492582,"logger":"Main","msg":"Setting up controller"}
{"level":"info","ts":1554124103.3493834,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"pod-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1554124121.0742912,"logger":"kubebuilder.webhook","msg":"installing webhook configuration in cluster"}
{"level":"info","ts":1554124121.0744402,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"pod-controller"}
{"level":"info","ts":1554124122.349403,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"pod-controller","worker count":1}
{"level":"info","ts":1554124122.5875223,"logger":"kubebuilder.webhook","msg":"starting the webhook server."}
{"level":"info","ts":1554124244.168139,"logger":"PoolManager","msg":"mac from pool was allocated to the pod","allocatedMac":"02:00:00:00:00:01","podName":"samplepod","podNamespace":"default"}
```